packages/web3-eth/src/web3_subscriptions.ts

```diff
@@ -16,7 +16,17 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 // eslint-disable-next-line max-classes-per-file
-import { BlockOutput, SyncOutput } from 'web3-common';
+import {
+	BlockInput,
+	BlockOutput,
+	LogsInput,
+	LogsOutput,
+	outputBlockFormatter,
+	outputLogFormatter,
+	outputSyncingFormatter,
```

Comment:
We should have consistent approach for results formatting across web3 4.x, 
so instead of using :
```
outputBlockFormatter,
	outputLogFormatter,
	outputSyncingFormatter
```
lets use web3.common [format](https://github.com/ChainSafe/web3.js/blob/acf0950ebcdc6ea0159175e9e8760420bd6aa8aa/packages/web3-common/src/formatter.ts#L248) function, like we are doing in eth package [eg](https://github.com/ChainSafe/web3.js/blob/acf0950ebcdc6ea0159175e9e8760420bd6aa8aa/packages/web3-eth/src/rpc_method_wrappers.ts#L243).

Reply:
Create new issue https://github.com/ChainSafe/web3.js/issues/5265